datalad/datalad-ebrains
# Template update

This PR brings the extension up to date with the extension template. There are these major changes:
- Switch from nose to pytest
- Update to the minimal requirements: Python 3.6 → 3.7, DataLad 0.16 → 0.17
- Update of Versioneer

Other changes are mostly related to boilerplate code.

FTR, this was done with `git merge --allow-unrelated-histories --squash`. I did it in two stages, to see how much conflict resolution needs to be redone when changes are squashed. I don't have a precise record, but I feel that I had to repeat resolving some changes (mostly rejections), but not all, the amount of conflicts on the second part was reduced, as the diff was smaller.

TODO:
- [x] Change tests to work again: currently test files still import nose, which fails
